.evergreen/download-mongodb.sh

```diff
@@ -40,18 +43,19 @@ get_distro ()
    echo $DISTRO
 }
 
-# get_mongodb_download_url_for "linux-distro-version-architecture" "latest|40|36|34|32|30|28|26|24"
-# Sets EXTRACT to aproprate extract command
-# Sets MONGODB_DOWNLOAD_URL to the aproprate download url
+# get_mongodb_download_url_for "linux-distro-version-architecture" "latest|44|42|40|36|34|32|30|28|26|24"
```

Comment:
Should this include "5.0"?

Also, looking at `case "$_VERSION" in` down below, the second argument (`_VERSION=$2`) should be dotted (e.g. "4.4").